Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/SqliteEamDb.java

```diff
@@ -1156,4 +1170,25 @@ private void releaseSharedLock() {
         rwLock.readLock().unlock();
     }
 
+    @Override
+    boolean doesColumnExist(Connection conn, String tableName, String columnName) throws SQLException {
+        final String tableInfoQueryTemplate = "PRAGMA table_info(%s)";  //NON-NLS
+        ResultSet resultSet;
+        Statement statement = conn.createStatement();
+        boolean columnExists = false;
+        resultSet = statement.executeQuery(String.format(tableInfoQueryTemplate, tableName));
```

Comment:
I think this should be in the try-catch block to make sure everything gets closed